basic/src/test/java/org/jboss/hal/testsuite/test/configuration/mail/MailSubsystemOperations.java

```diff
@@ -0,0 +1,36 @@
+package org.jboss.hal.testsuite.test.configuration.mail;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.jboss.hal.testsuite.creaper.command.AddSocketBinding;
+import org.jboss.hal.testsuite.util.AvailablePortFinder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+
+import java.io.IOException;
+
+class MailSubsystemOperations {
+
+    private static final Logger log = LoggerFactory.getLogger(MailSubsystemOperations.class);
+
+    private final OnlineManagementClient client;
+
+    MailSubsystemOperations(OnlineManagementClient client) {
+        this.client = client;
+    }
+
+    String createSocketBinding() throws IOException, CommandFailedException {
+        String socketBinding = "mailTestCaseSb_" + RandomStringUtils.randomAlphanumeric(6);
+        return createSocketBinding(socketBinding);
+    }
+
+    String createSocketBinding(String name) throws IOException, CommandFailedException {
```

Comment:
Is it really necessary to have both variants of the `createSocketBinding` method? IMHO it makes the code less logic and readable. IMHO `String createSocketBinding()` should be sufficient. Just suggestion, I do not insist on such unification. WDYT?